includes/class-sensei-lesson.php

```diff
@@ -280,6 +280,21 @@ public function meta_box_setup() {
 		// Add Meta Box for Lesson Preview
 		add_meta_box( 'lesson-preview', esc_html__( 'Preview', 'sensei-lms' ), array( $this, 'lesson_preview_meta_box_content' ), $this->token, 'side', 'low' );
 
+		/**
+		 * Filters the content drip metabox toggle.
+		 *
+		 * @hook  sensei_lesson_content_drip_hide
+		 * @since 4.1.0
+		 *
+		 * @param  {bool} $hide_content_drip
+		 * @return {bool}
```

Comment:
Could we add descriptions for the parameters so they'll show up in our [API docs](https://automattic.github.io/sensei/)?

Reply:
Yep - added in 413f9ebb0c81233cdef15e15db556af33d36f740.

Reply:
Cool. The return variable could also use a description though. 😉

Reply:
Note to self: everything needs a description! 😄

Thanks, fixed in 25c241f1fe5715052d0d64bc7972cbfc11a14485. 🙂